toolbox/gui/panel_protocols.m

```diff
@@ -1209,16 +1209,16 @@ function MarkUniqueNode( bstNode ) %#ok<DEFNU>
     % File info is not passed in input
     sqlConn = sql_connect();
     if nargin < 5 || isempty(sSubjectTargetRaw)
-        iSubjectTargetRaw = db_get('SubjectFromFunctionalFile', sqlConn, srcFile);
-        sSubjectTargetRaw = db_get('Subject', sqlConn, iSubjectTargetRaw);
+        iSubjectTargetRaw = db_get(sqlConn, 'SubjectFromFunctionalFile', srcFile);
+        sSubjectTargetRaw = db_get(sqlConn, 'Subject', iSubjectTargetRaw);
     end
     if nargin < 6
         ParentFile = [];
     end
     isAnatomy = ismember(srcType, {'anatomy','cortex','scalp','innerskull','outerskull','fibers','fem','other'});
     % Get source subject
     if ~isAnatomy
-        iSubjectSrcRaw = db_get('SubjectFromStudy', sqlConn, iSrcStudy);
+        iSubjectSrcRaw = db_get(sqlConn, 'SubjectFromStudy', iSrcStudy);
         sSubjectSrcRaw = db_get('Subject', iSubjectSrcRaw);

```

Comment:
You might as well re-use the same sqlConn as well here, as the previous line.

Reply:
Yes, I'll aim to use the same sqlConn for serial calls to db_get and db_set